osidb/models.py

```diff
@@ -2092,6 +2092,10 @@ def is_community(self) -> bool:
     #      - see e.g. backends/jira/rest/request.py in SFM2
     # TODO unlike SFM2, SLA can't (?) evaluate the new streams part as long as this is a property
     #      on affect (since affect doesn't have streams). Should we file a new Jira to fix it?
+    # TODO: There's probably an omission in not creating a compliance-priority tracker label IN OSIDB.
+    #       See backends/jira/rest/request.py in SFM2. If that's the case, file a Jira for OSIDB.
+    # TODO: There's probably an omission in not creating a contract-priority tracker label IN SFM2.
```

Comment:
@osoukup There's probably an omission in not creating a contract-priority tracker label **_IN SFM2_**. OSIDB creates that label, but can't find it in SFM2. If that's the case, shall we file a Jira for SFM2?
(Found this while looking at the SFM2 changes that are the blueprint for this PR.)

Reply:
Facepalm. I looked at an outdated version of SFM2. Thanks!